e2e/artefacts/k8s/pod-nf.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfpod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: nfnet, nfnet
+spec:
+  hostNetwork: true
+  containers:
+  - name: nf 
+    image: intel-networkfunction:latest
+    imagePullPolicy: IfNotPresent
+    securityContext:
+      privileged: true
+    command: [ "/app/nf"]
+    resources:
+      requests:
+        intel.com/ipu: '1'
```

Comment:
Should be 2 here - as we attach two APFs per NF

Reply:
> Should be 2 here - as we attach two APFs per NF. Also 2 for the limits label

Yes, it should 2. I will update it.